doc/sphinx/language/extensions/evars.rst

```diff
@@ -77,20 +77,21 @@ guess the missing piece of information.
 Automatic resolution of existential variables
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Existential variables that correspond to (shelved) goals that are most often
-resolved by a side effect, i.e., they get instantiated during the manipulation
-of other goals that depend on them.
-
-Some tactics automatically resolve goals that are created when existential
-variables are created.
+Existential variables that are used in other goals are generally resolved
+automatically as a side effect of other tactics.
 
 .. _automatic-evar-resolution:
 
 .. example:: Automatic resolution of existential variables
 
+   :n:`?x` and :n:`?m` are used in other goals.  The :tacn:`exact`
```

Comment:
"The" in "The `exact`" means a reference to a specific `exact`.  Without the article it would refer to `exact` in general, i.e. all uses of `exact`.  Right?  But no problem changing if you prefer.

Reply:
Yes, but maybe not completely obvious to non-anglophones (not sure).